migaloo/assetlist.json

```diff
@@ -382,6 +382,52 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/lsdshark.png"
         }
       ]
+    },
+    {
+      "description": "The native staking token of Terra.",
+      "denom_units": [
+        {
+          "denom": "ibc/4627AD2524E3E0523047E35BB76CC90E37D9D57ACF14F0FCBCEB2480705F3CB8",
+          "exponent": 0
+        },
+        {
+          "denom": "luna",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/4627AD2524E3E0523047E35BB76CC90E37D9D57ACF14F0FCBCEB2480705F3CB8",
+      "name": "Luna",
+      "display": "luna",
+      "symbol": "LUNA",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "terra2",
+            "base_denom": "uluna",
+            "channel_id": "channel-86"
```

Comment:
Pending registration of this IBC connection. The transfer<>transfer channel should be first in the array of channels, then the wasm connetion after.

Reply:
There is no Transfer-Transfer channel on this connection...

Reply:
wait - this literally _is_ a transfer-transfer channel. i don't understand what the problem is right now... can you please elaborate? please note: this is migaloo-terra2 and _NOT_ migaloo-terra

Reply:
My bad, this is already registered. I had terra in mind